tuning/automation/GenerateTuningConfigurations.py

```diff
@@ -397,7 +400,7 @@ def OutputScript2(problemMapper, scriptPath, namePart):
         with open(outputFileName, 'w') as f:
             for line in lines:
                 if "rocblas-bench" in line:
-                    f.write("ROCBLAS_TENSILE_LIBPATH=${TENSILE_LIBRARY} %s\n" % line)
+                    f.write("ROCBLAS_TENSILE_LIBPATH=${TENSILE_LIBRARY} %s0" % line)
```

Comment:
If you look at ExtractSizes.py in the function: 
def GetRocBLASParser(): 

at the line :
    lineParser.add_argument("-i",dest="i", type=int,default=1)

Try setting the default to 10

Reply:
Thanks for the suggestion Bill. I've made that change in my latest commit.